README.md

```diff
@@ -1,51 +1,36 @@
 # RoboSaver
 
-RoboSaver turns your Gnosis Pay card into an automated savings account!
+RoboSaver turns your [Gnosis Pay card](https://gnosispay.com/) into an automated savings account!
 
-Unused EURE on your card gets deposited into a liquidity pool, where it collects yield and swapping fees. As soon as your card's balance gets below a certain threshold, the RoboSaver will withdraw some EURE from the pool for you and top up your card. Thus creating the perfect balance between having EURE ready for spending and putting EURE to work!
+Unused [EURe](https://monerium.com/tokens/) on your card gets deposited into a liquidity pool, where it collects yield and swapping fees. As soon as your card's balance gets below a certain threshold, the RoboSaver will withdraw some EURe from the pool for you and top up your card. Thus creating the perfect balance between having EURe ready for spending and putting EURe to work!
 
-![](diagram.drawio.png)
+<p align="center"><img src="diagram.drawio.png"></p>
```

Comment:
nice trick to center the img!

wonder if it will be more appropriate to use instead a `<div>` tag

Reply:
according to [this thread](https://stackoverflow.com/questions/70909227/why-cant-i-center-markdown-image-syntax-with-p-tag-but-div-works) there will still be an automatic insertion of a `<p>` tag when using a `<div>`, so that feels a bit redundant.

Reply:
interesting i was not aware of that nuance